fsevents.js

```diff
@@ -6,6 +6,13 @@
 /* jshint node:true */
 'use strict';
 
+if (process.platform !== 'darwin') {
+  var err = new Error('Cannot find module \'fsevents\'');
```

Comment:
I've pushed a new commit.

Was a bit much trying to cram into a `node -e`, so there's a new `install.js`, and `scripts.install` is now `node install`